doc/source/f2py/code/ftype.f

```diff
@@ -4,6 +4,6 @@ SUBROUTINE FOO(N)
 Cf2py integer optional,intent(in) :: n = 13
       REAL A,X
       COMMON /DATA/ A,X(3)
-      PRINT*, "IN FOO: N=",N," A=",A," X=[",X(1),X(2),X(3),"]"
+C     PRINT*, "IN FOO: N=",N," A=",A," X=[",X(1),X(2),X(3),"]"
```

Comment:
Maybe we want to remove this line entirely?

Reply:
Ack, sorry, didn't see this earlier, I'll open a general cleanup to make sure we have no print statements in the `f2py` tests this weekend.